health/health.d/softnet.conf

```diff
@@ -36,5 +39,7 @@
    every: 10s
     warn: $this > (($status >= $WARNING) ? (0) : (10))
    delay: down 1h multiplier 1.5 max 2h
-    info: average number of drops in the last 1min, because sysctl net.route.netisr_maxqlen was exceeded (this can be a cause for dropped packets)
+    info: average number of drops in the last minute \
+          due to sysctl net.route.netisr_maxqlen was exceeded \
+          (this can be a cause for dropped packets)
```

Comment:
```suggestion
    info: average number of drops in the last minute \
          due to exceeded sysctl net.route.netisr_maxqlen \
          (this can be a cause for dropped packets)
```